cmd/soroban-cli/src/commands/events.rs

```diff
@@ -43,7 +43,7 @@ pub struct Cmd {
     /// the command-line today, though that set can have multiple IDs/topics.
     #[arg(
         long = "id",
-        num_args = 1..=5,
+        num_args = 1..=6,
```

Comment:
?

Reply:
The docs above say the max is 5 and ..6 means at most 5. Unless the comment is wrong